services/actions/notifier.go

```diff
@@ -101,6 +101,118 @@ func (n *actionsNotifier) IssueChangeStatus(ctx context.Context, doer *user_mode
 		Notify(ctx)
 }
 
+// IssueChangeAssignee notifies assigned or unassigned to notifiers
+func (n *actionsNotifier) IssueChangeAssignee(ctx context.Context, doer *user_model.User, issue *issues_model.Issue, assignee *user_model.User, removed bool, comment *issues_model.Comment) {
+	ctx = withMethod(ctx, "IssueChangeAssignee")
+
+	var err error
+	if err = issue.LoadRepo(ctx); err != nil {
+		log.Error("LoadRepo: %v", err)
+		return
+	}
+
+	if err = issue.LoadPoster(ctx); err != nil {
+		log.Error("LoadPoster: %v", err)
+		return
+	}
+
+	permission, _ := access_model.GetUserRepoPermission(ctx, issue.Repo, issue.Poster)
```

Comment:
Could move this below `if issue.IsPull` because it's only needed for issues. (Why is that the case? Why has the pull request `AccessModeNone` but the issue has the real permissions?)

Reply:
> Why has the pull request AccessModeNone but the issue has the real permissions?

I have no idea about this now. I think we can ask @wolfogre, as it comes from the origin implement.

Reply:
Whether all the concerns are resolved? @KN4CK3R @yp05327

Reply:
The `permission` is used to fill the payload to allow scripts in actions to check if they can push or pull the repo. It inherits the design of webhooks, I don't think the actions scripts really read it, so it should be OK to use the same `permission` for both issues and PRs. It doesn't affect the actual permissions of actions scripts.